src/Shared/Logger/infra/file_handler.go

```diff
@@ -0,0 +1,73 @@
+package infra
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+)
+
+type FileData struct {
+	handle *os.File
+	values []ValueTimestamp
+}
+
+func NewFile(parentPath string, name string) *FileData {
+	file, err := os.Create(path.Join(parentPath, fmt.Sprintf("%s.csv", name)))

```

Comment:
Why do we use .csv?

Reply:
We spoke to electromagnetics and agreed on that format. They can import it in matlab easily.